src/Auth/OAuth/Paypal.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Auth\OAuth;
+
+use Auth\OAuth;
+
+// Reference Material
+// https://developer.paypal.com/docs/api/overview/
+
+class Paypal extends OAuth
+{
+    /**
+     * @var string
+     */
+    private $endpoint= [
+        'sandbox' => 'https://www.sandbox.paypal.com/',
+        'live' => 'https://www.paypal.com/',
+    ];
+
+    private $resourceEndpoint = [
+        'sandbox' => 'https://api.sandbox.paypal.com/v1/',
+        'live' => 'https://api.paypal.com/v1/',
+    ];
+
+    private $environment = 'sandbox';
```

Comment:
As we currently don’t have any support for multiple environments we should probably have the live one as default.

Reply:
Changed environment to live. Shall I remove entirely the sandbox?